src/java.base/share/classes/java/lang/String.java

```diff
@@ -248,8 +249,10 @@
      * an empty character sequence.  Note that use of this constructor is
      * unnecessary since Strings are immutable.
      */
+
```

Comment:
Remove superfluous empty line.

Reply:
I have removed the empty line in my new commit